bundle/jsky.app.ot/src/main/scala/jsky/app/ot/ags/BagsManager.scala

```diff
@@ -162,7 +164,7 @@ object BagsState {
         if (r.posAngle === ctx.getPositionAngle) hash
         else hashObs(ctx.withPositionAngle(r.posAngle))
       }
-      (IdleState(k, Some(h)), BagsManager.applyAction(k, results))
+      (IdleState(k, Some(h)), BagsManager.applyAction(k, Some(hash), results))
```

Comment:
`hash` is the AGS computation made from the relevant bits of an observation such as it was before any updates are applied.